packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -35,7 +37,10 @@ export const GridFilterToolbarButton: React.FC<{}> = () => {
           {activeFilters.map((item) => ({
             ...(lookup[item.columnField!] && (
               <li key={item.id}>
-                {lookup[item.columnField!].headerName || item.columnField} {item.operatorValue}{' '}
+                {lookup[item.columnField!].headerName || item.columnField}{' '}
```

Comment:
With the spaces, it might be better to use a string template

Reply:
I agree string templates are better and cleaner but some editors might trim the trailing spaces. I did test it and it is working. I'll update it and see if there will be an issue with it.